### PR TITLE
feat: add PhysicalMachineChaos UI

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What is changed and how it works?

What's Changed:

This PR allows the manipulation of `PhysicalMachineChaos` experiments, schedules, and workflows in the dashboard interface.

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

File: ui/src/components/NewExperimentNext/data/types.ts [hunk 1]
```diff
@@ -534,14 +541,430 @@ const data: Record<Kind, Definition> = {
...
 
-const AwsChaosCommonSchema = Yup.object({
+const networkPhysicCommon: Spec = {
+  correlation: {
+    field: 'text',
+    label: 'Correlation',
+    value: '',
+    helperText: 'The correlation of corrupt',
+  },
+  device: {
```
Comment (iguoyr) [inline ui/src/components/NewExperimentNext/data/types.ts:551]: @WangXiangUSTC `device` is a required field, it needs to be validated by webhook.

File: ui/src/components/NewExperimentNext/data/types.ts [hunk 2]
```diff
@@ -534,14 +541,430 @@ const data: Record<Kind, Definition> = {
...
+  },
+}
+
+const diskPhysicCommon: Spec = {
+  size: {
+    field: 'text',
+    label: 'Size',
+    value: '',
+    helperText:
+      'The supported formats of the size are: c(=1), w(=2), kB(=1000), K(=1024), MB(=1024), M(=1024x1024), GB and so on.',
```
Comment (iguoyr) [inline ui/src/components/NewExperimentNext/data/types.ts:606]: > MB(=1024)

0.0

File: ui/src/components/NewExperimentNext/data/types.ts [hunk 3]
```diff
@@ -534,14 +541,430 @@ const data: Record<Kind, Definition> = {
...
+  },
+  path: {
+    field: 'text',
+    label: 'Path',
+    value: '',
+    helperText: 'Specify the file path of the data to be read/written',
```
Comment (iguoyr) [inline ui/src/components/NewExperimentNext/data/types.ts:612]: @WangXiangUSTC Dashboard could only get events which is like `HTTP status is not ok` now, If I want to see more information about this error, I need to check logs on the physical machine. I wonder if we could return the error information to chaos-controller-manager, and then we can see more detailed events from dashboard.

**Comment by WangXiangUSTC** [comment]
@iguoyr Thanks, since this doesn't have much to do with UI, I will update them in another pr.

**Review by iguoyr — APPROVED** [review]
LGTM

File: ui/src/components/NewExperimentNext/data/types.ts (hunk 2 above)
Comment (g1eny0ung) [reply ui/src/components/NewExperimentNext/data/types.ts:606]: Very messed here. 😵 Will update all mentioned descriptions in a new PR. Good catch!

**Pull request merged**